src/platformdirs/windows.py

```diff
@@ -24,7 +24,11 @@ def user_data_dir(self) -> str:
          ``%USERPROFILE%\\AppData\\Roaming\\$appauthor\\$appname`` (roaming)
         """
         const = "CSIDL_APPDATA" if self.roaming else "CSIDL_LOCAL_APPDATA"
-        path = os.path.normpath(get_win_folder(const))
+        path = get_win_folder(const)
+        if path is None:
+            raise ValueError(f"No path found for CSIDL: {const}")
```

Comment:
This before never could be None, how come now it can?

Reply:
There is no environment variable for the Documents folder, so the `get_win_folder_from_env_vars` function has to return None for the documents folder; the None carries over to `get_win_folder`.  Returning the None is so a default can be used rather than just raising an error in a normal situation.

Reply:
Please revert this to the previous state of raising and not checking here.

Reply:
What should I do about the Documents folder when `get_win_folder_from_env_vars` is used then?